types/leaflet-routing-machine/leaflet-routing-machine-tests.ts

```diff
@@ -7,9 +7,24 @@ L.tileLayer('http://{s}.tile.osm.org/{z}/{x}/{y}.png', {
     attribution: '&copy; <a href="http://osm.org/copyright">OpenStreetMap</a> contributors'
 }).addTo(map);
 
+const waypoints = [
+    L.latLng(57.75934, 11.93927),
+    L.latLng(57.74, 11.94),
+    L.latLng(57.6792, 11.949),
+];
+
+const myPlan = new L.Routing.Plan(waypoints, {
+    addWaypoints: false,
+    createMarker: (index: number, waypoint: L.Routing.Waypoint, numberWaypoints: number) => {
+      if (index === 0 || index === numberWaypoints - 1) {
```

Comment:
Thanks for updating the definition. Do you think this is a good example of using `createMarker` function? What if we update this function as,
```
if(waypoint && index > -1 && index < numberOfWaypoints){
    return L.marker(waypoint.latLng);
}
return false;
```
Please share your opinion.

Reply:
That's works as well! I will update the function.

Reply:
Awesome. Thanks